neutromeratio/ani.py

```diff
@@ -19,96 +19,61 @@ class ANI1_force_and_energy(object):
     """
 
     def __init__(self, 
-                device:torch.device, 
                 model:torchani.models.ANI1ccx, 
-                atom_list:list, 
-                platform:str, 
-                tautomer_transformation:dict={},
+                atoms:str, 
                 ):
         
         self.device = device
         self.model = model
-        self.species = model.species_to_tensor(atom_list).to(device).unsqueeze(0)
-        self.atom_list = atom_list
+        self.atoms = atoms
+        self.species = self.model.species_to_tensor(atoms).to(device).unsqueeze(0)
         self.platform = platform
-        self.lambda_value = 1.0
-        self.bias_harmonic = []
-        self.bias_flat_bottom = []
-        self.bias_applied = []
-        self.tautomer_transformation = tautomer_transformation
-        self.restrain_acceptor = False
-        self.restrain_donor = False
-
+        self.use_pure_ani1ccx = False
+        
+        self.flat_bottom_restraint = False
```

Comment:
instead of having flags for `self.flat_bottom_restraint` and `self.harmonic_restraint`, could we have only a `self.list_of_restraints`?